### DISCUSSION TITLE
replace momentjs with something more suitable

### DISCUSSION THREAD

**Discussion by haexhub** [ConduitPlatform/Conduit#849] [General]
Hi Conduit Team.
Your are using momentjs in your conduit-ui. This library is in maintenance mode and should not be used anymore.
I would propose to change this with luxon or dayjs or anything you like and find more suitable.

**Comment by kkopanidis** [comment]
Hi @haexhub, we're aware of this but moment is very reliable and we're not missing any features that another library might provide. We will replace it eventually, but it is not currently a priority.
